src/types/package.d.ts

```diff
@@ -0,0 +1,39 @@
+interface Wallet {
```

Comment:
We should rename this to `ManifestWalletProperty` to better indicate what it is.